### DISCUSSION TITLE
UI Design - More Brushed Metal

### DISCUSSION THREAD

**Discussion by iamhumanipromise** [helloSystem/hello#361] [Ideas]
There is something that is missing from the look of HelloSystem and no, it isn’t the fact that it isn’t part of the PureDarwin project and running on Darwin 

(Though that is another issue, hopefully they can borrow from and contribute to the work here so both projects can thrive!)

There is potential for HelloSystem to be a modern re-imagining of elements of Platinum, Aqua from 10.2 and the brushed metal look of 10.4, along with some modern effects and transparency borrowed from QT and GTK.

**Comment by probonopd** [comment]
Any help appreciated :-) helloSystem is using Qt, and making a brushed metal Qt theme should be doable.
But keep in mind that helloSystem is not intended to be a visual clone of anything. It's about how it works that should be welcoming to switchers from the Mac, we are not aiming at being visually very similar.

**Reply by Calypso314** [reply]
On that note, we can even give people more options? Brushed metal is your thing? Sure, with a click your UI gets that look. Want frosted glass? Here you go. 

If I might add, yes, we should not copy Apple. We only should be driven by the same force that once used to drive Apple.

I think I can help in graphic design department. I will suggest few of the options when they are ready, if they are accepted fine, if not, oh well.  I only today discovered that helloSystem exists and I am already liking what I am seeing, I am gonna install it in my VM right now.

**Reply by probonopd** [reply]
Hello @Calypso314. Are you familiar with how Qt works, and how Qt themes and Qt stylesheets work? If no, now might be a good time to look into it ;-)

**Reply by Calypso314** [reply]
> Hello @Calypso314. Are you familiar with how Qt works, and how Qt themes and Qt stylesheets work? If no, now might be a good time to look into it ;-)

I downloaded Qt Designer yesterday. I also am working on icon design in GIMP (icons for file types).

Hear me out, I am still unsure about which design language are we gonna follow. Do we want 3D elements, with some gloss? Do we want flat elements? Do we want something in between? Maybe neither of those but something completely else? I have so many ideas in my mind for many different things, but I hope people can decide on which route should we take.

In my humble opinion we should strive for consistency. I hate inconsistency and things that just don't fit together. Once we decide which path should we take then we will be able to be consistent in our design.

**Reply by probonopd** [reply]
Definitely no "flat design". 

imho Mac OS X Aqua design peaked around 10.4. We want to provide an experience where a switcher from the Mac would feel "at home" while it should look differently enough as to not being mistaken for a "clone". 

Personally I like the style of this icon very much:

![image](https://user-images.githubusercontent.com/2480569/176237384-044ebe9a-394f-4588-802f-eb4c2d648c0c.png)

It is definitely not "flat". It does have some level of skeucomorphism but not overdone. It has depth, shine, shadows, and reflections but not in an excessive way. It has perspective that could become formalized (similar to how perspective is used in Haiku OS icons).

![image](https://user-images.githubusercontent.com/2480569/176238390-2a478bda-4ffd-4bc7-9f67-65f79c02b2f7.png)

**Comment by louies0623** [comment]
Hey @Calypso314  Can you make the material of this file into a Qt style theme? pls thank.
I try to get someone else to help but he doesn't have time.

[Theme.pdf](https://github.com/helloSystem/hello/files/10176142/Theme.pdf)
